source/fundamentals/connection.txt

```diff
@@ -55,10 +55,26 @@ options. In the example, we set two connection options:
 ``maxPoolSize=20`` and ``w=majority``. To learn more about connection
 options, read the :ref:`golang-connection-options` section of this guide.
 
-The following code shows how you can use a connection string in a
-client to connect to MongoDB. In this example, the client uses 
-the :ref:`Stable API <golang-stable-api>` feature, though
-you can connect to MongoDB without this feature.
+Create a Client to Connect to MongoDB
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You need to create a client to connect to MongoDB, manage your
+connection, and run database commands. You can create a client by
+passing options to the ``Connect()`` method. These options
+include your connection string and any other client options.
```

Comment:
Issue:

I think there's ambiguity in the current presentation of "options" that might only be specific to the Golang driver.

In the driver, there's an "option" package from which you access the Client and include your connection string via ApplyURI(). However, the general term "option" describes something that you can omit (and doesn't have a default value) and have it function properly. I think the Client() contains a default value (mongodb://localhost:27017) when you just instantiate with "mongo.NewClient()". This allows you to connect locally, but it would be good to identify as required in all cases except when you're connecting locally to the default port, especially since the prior content describes the connection string in detail. 

Suggestion:

You can create a client by calling the mongo.NewClient() method. You can specify your connection string and other connection options using the options package. To specify your connection string, call the method ApplyURI() on the options package. To specify <other options such as... > call < other methods >. Pass your options to the Connect() method to connect to the MongoDB deployment that you specified in your connection string.